smart-contracts/test/UnlockProtocolToken/governor.js

```diff
@@ -12,15 +12,15 @@ const { getEvent } = require('@unlock-protocol/hardhat-helpers')
 
 const PROPOSER_ROLE = ethers.keccak256(ethers.toUtf8Bytes('PROPOSER_ROLE'))
 
-describe('UnlockProtocolGovernor', () => {
+// default values
+const SIX_DAYS = 43200 // in blocks
```

Comment:
hum, I think we should not use blocks because that's different based on what chain we go for. Can we use seconds rather?

Reply:
Just moved the gov and erc20 votes to use timestamps instead of blocks